beacon-chain/core/helpers/rewards_penalties.go

```diff
@@ -20,33 +20,18 @@ var totalActiveBalanceCache = cache.NewActiveBalanceCache()
 //    Return the combined effective balance of the ``indices``. (1 Gwei minimum to avoid divisions by zero.)
 //    """
 //    return max(sum([state.validator_registry[index].effective_balance for index in indices]), 1)
-func TotalBalance(state *pb.BeaconState, indices []uint64) (uint64, error) {
-	epoch := CurrentEpoch(state)
-	total, err := totalBalanceCache.TotalBalanceInEpoch(epoch)
```

Comment:
Removing total balance cache usage for `TotalBalance`. It was not implemented correctly and it's not needed. The input should have been `indices` not `epoch`